app/models/XPath.scala

```diff
@@ -32,47 +32,46 @@ case class XPath(value: String) {
-      case x if x.matches("^Consignment/TransportEquipment(.+)$")                                                          => TransportDetails
-      case x if x.matches("^Consignment/ActiveBorderTransportMeans(.+)$")                                                  => TransportDetails
-      case x if x.matches("^Consignment/PlaceOfLoading(.+)$")                                                              => RouteDetails
-      case x if x.matches("^Consignment/PlaceOfUnloading(.+)$")                                                            => RouteDetails
-      case x if x.matches("^Consignment/PreviousDocument(.+)$")                                                            => Documents
-      case x if x.matches("^Consignment/SupportingDocument(.+)$")                                                          => Documents
-      case x if x.matches("^Consignment/TransportDocument(.+)$")                                                           => Documents
-      case x if x.matches("^Consignment/HouseConsignment(\\[[\\d]*])/ConsignmentItem(\\[[\\d]*])/PreviousDocument(.+)$")   => Documents
-      case x if x.matches("^Consignment/HouseConsignment(\\[[\\d]*])/ConsignmentItem(\\[[\\d]*])/SupportingDocument(.+)$") => Documents
-      case x if x.matches("^Consignment/HouseConsignment(\\[[\\d]*])/ConsignmentItem(\\[[\\d]*])/TransportDocument(.+)$")  => Documents
-      case x if x.matches("^Consignment/countryOfDispatch$")                                                               => TransportDetails
-      case x if x.matches("^Consignment/countryOfDestination$")                                                            => TransportDetails
-      case x if x.matches("^Consignment/containerIndicator$")                                                              => TransportDetails
-      case x if x.matches("^Consignment/inlandModeOfTransport$")                                                           => TransportDetails
-      case x if x.matches("^Consignment/modeOfTransportAtTheBorder$")                                                      => TransportDetails
-      case x if x.matches("^Consignment/referenceNumberUCR$")                                                              => TransportDetails
-      case x if x.matches("^Consignment/CountryOfRoutingOfConsignment(.+)$")                                               => RouteDetails
-      case x if x.matches("^Consignment/TransportCharges(.+)$")                                                            => TransportDetails
-      case x if x.matches("^Consignment/HouseConsignment(\\[[\\d]*])/ConsignmentItem(.+)$")                                => Items
-
+    val pf: PartialFunction[String, Task] = {
+      case x if x.matches("^(.*)/TransitOperation/declarationType$")                                                            => PreTaskList
```

Comment:
is it (.*) at the beginning now - because you deleted the replace thing?

Reply:
@joecefnhedog yep the problem was two-fold

The error pointer after an IE013 rejection would be something like `/CC013C/Consignment/...`

Also, we saw an error pointer in QA like `//Consignment/...`

This updated logic ensures we cover a much wider range of scenarios

Reply:
Looks safer to me 👍